Dockerfile

```diff
@@ -62,6 +62,8 @@ RUN set -eux; \
     docker-php-ext-enable apcu; \
     pecl install redis \
     && docker-php-ext-enable redis; \
+    pecl install xdebug \
```

Comment:
This is the container that we ship, I think this approach is good - but it should go into the `fpm-dev` container since it isn't something we'd want to have in our production environments.

Reply:
good catch, i moved it around. :heavy_check_mark:

Reply:
I believe you, but I'm not seeing that here. Push again? Maybe GitHub is slow today?

Reply:
hah, my bad. committed and pushed now.